MobileSyncExplorerReactNative/js/ContactScreen.js

```diff
@@ -107,11 +107,11 @@ class ContactScreen extends React.Component {
         var errorMessage = null;
         const lastError = this.state.contact.__last_error__;
         if (lastError) {
-            try {
+            try {                
                 if (Platform.OS == 'ios') {
-                    errorMessage = new RegExp('.*message = "([^"]*)".*', 'm').exec(lastError)[1];
```

Comment:
Last error returned on iOS is no longer a string we need to parse, however it's not quite the same as android :-(